fearless/Modules/AccountImport/Model/MnemonicTextNormalizer.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+import SoraFoundation
+
+struct MnemonicTextNormalizer: TextProcessing {
+    func process(text: String) -> String {
+        text
+            .replacingOccurrences(of: "\t", with: "")
```

Comment:
I think tabulation is in CharacterSet.whitespacesAndNewlines. Can we get rid of this line?